src/cmd/tools/zarf.go

```diff
@@ -155,31 +155,17 @@ var updateCredsCmd = &cobra.Command{
 
 			// Update artifact token (if internal)
 			if slices.Contains(args, message.ArtifactKey) && newState.ArtifactServer.PushToken == "" && newState.ArtifactServer.IsInternal() {
-				tunnel, err := c.NewTunnel(cluster.ZarfNamespaceName, cluster.SvcResource, cluster.ZarfGitServerName, "", 0, cluster.ZarfGitServerPort)
-				if err != nil {
-					return err
-				}
-				_, err = tunnel.Connect(cmd.Context())
-				if err != nil {
-					return err
+				pushToken, err := c.UpdateInternalArtifactServerToken(ctx, oldState.GitServer)
+				updateErr := fmt.Errorf("unable to create the new Gitea artifact token: %w", err)
+				// TODO once Zarf is changed so the default state is empty for a service when it is not deployed
+				// and sufficient time has passed for users state to get updated we should error on isNotFound
+				if err != nil && !kerrors.IsNotFound(err) {
```

Comment:
> Check if err 
If err 
>>  Check if is not found 
return  not found 
> return `newState.ArtifactServer.PushToken = pushToken` , no else

Reply:
I think the formatting got messed up here

Reply:
fixed (hopefully that makes more sense)

Reply:
I wanted to not return the errors on `kerrors.IsNotFound` since some users might not have a git server deployed and in that case this function would always error. 

I just changed the code to not get to this block if the git server doesn't exist which hopefully clarifies intent